src/cborvalidation.c

```diff
@@ -33,7 +33,7 @@
 #endif
 
 #include "tinycbor/cbor.h"
-#include "tinycbor/cborinternal_p.h"
+#include "tinycbor/cborconstnats_p.h"
```

Comment:
This should be `cborconstants_p.h`

Reply:
Bummer. Compilation has not shown errors as the files does not seem to be used by our samples.